test/e2e/homeworkspaces/home_workspaces_test.go

```diff
@@ -91,7 +91,15 @@ func TestUserHomeWorkspaces(t *testing.T) {
 		t.Run(testCase.name, func(t *testing.T) {
 			t.Parallel()
 			tokenAuthFile := framework.WriteTokenAuthFile(t)
-			server := framework.PrivateKcpServer(t, framework.WithCustomArguments(append(framework.TestServerArgsWithTokenAuthFile(tokenAuthFile), testCase.kcpArgs...)...))
```

Comment:
We're not using the directories to determine the location of the auth file, is that on purpose? Why does this test need custom dirs?

Reply:
Yeah you're right, it's only really needed by `TestAuditLogs` - updated and I'll retitle the PR, thanks!